platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbStatistics.java

```diff
@@ -60,27 +60,27 @@ public class MerkleDbStatistics {
     private SpeedometerMetric leafByKeyReadsPerSecond;
     private SpeedometerMetric leafByPathReadsPerSecond;
 
-    private IntegerGauge internalHashesStoreFileCount;
-    private DoubleGauge internalHashesStoreTotalFileSizeInMB;
+    private IntegerGauge hashesStoreFileCount;
```

Comment:
If you can avoid changes to this class, I would appreciate it :) These metrics (method names, metric names, metric type - basically, all the class) are changed in one of my upcoming fixes a lot. You can keep using old names here, and I'll make name changes (e.g. "internalHashes" -> "hashes") in my PR. If not, I'll resolve the conflicts in my branch, not a big deal

Reply:
I'm fine with reverting this change as long as you can rename it later.